Resources/Scripts/Import-ApimOpenApiDefinitionFromFile.ps1

```diff
@@ -40,16 +42,17 @@ try {
     Write-Host "Retrieving ApiId for API $ApiName"
     $Api = Get-AzApiManagementApi -Context $Context -ApiId $ApiName
```

Comment:
Is this needed now?  The parameters passed into Import-AzApiManagementApi all come from the param block of the script

Reply:
Probably not, is the answer